src/components/transaction-nft-finish.tsx

```diff
@@ -110,7 +111,18 @@ export const TransactionNftFinish = ({
 
       <NetworkFee fee={fee.expectedFee} />
 
-      <Spacer />
+      <View style={styles.providerContainer}>
+        <Text
+          variant={TextVariant.t14}
+          color={Color.textBase2}
+          children={app.provider.name}
```

Comment:
Do not pass children as props. Instead, nest children between the opening and closing tags.sonarlint(typescript:S6748)